android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=zulip/zulip-mobile
+defaults.org=zulip
+defaults.project=zulip-mobile
+auth.token=getTokenFrom https://sentry.io/api/
```

Comment:
Is there a way to read that from external file? Or will we need to replace manually before build? Maybe replace with something easier to search for like SENTRY-API-TOKEN-HERE